pcp_alpha/Backend/pcp_app/tests.py

```diff
@@ -353,3 +360,65 @@ def test_validate_form(self, rf):
         url_var = "/action/val_target_form/"
         status_obj = self.status_code_test(url_str=url_var, function_obj=val_target_form, rf=rf)
         assert status_obj.status_code == 200
+
+    def test_render_edit_target_form(self, rf):
+        """
+        This test checks if it renders the
+        form correctly to edit targets.
+        """
+        target_key = ""
+        for target_item in get_brain_targets():
+            target_key = target_item["id"]
+        url_var = "/edit_target_form/{}".format(target_key)
+        status_obj = self.status_code_test(url_str=url_var, function_obj=edit_target_form, rf=rf, target_id=target_key)
+        assert status_obj.status_code == 200
+
+    @staticmethod
+    def test_edit_target_form():
+        """
+        This test is imitating editing a new target and
+        updating it back to Brain.Targets.
+        """
+        target_key = ""
+        for target_item in get_brain_targets():
+            target_key = target_item["id"]
+
+        if check_dev_env() is not None:
```

Comment:
We can probably take these check_dev_env() statements out now right (since pytest won't be running in prod)? Or is there another reason to keep them in?